layers/+tools/docker/README.org

```diff
@@ -28,6 +30,28 @@ file.
 You will also need the native package [[https://www.docker.com/][Docker]] for the actual container management.
 For syntax checking the external library =hadolint= is also necessary.
 
+** LSP
+This layer can be used with [[https://github.com/rcjsuen/dockerfile-language-server-nodejs][=dockerfile-language-server-nodejs=]] and emacs
+[[https://github.com/emacs-lsp/lsp-mode][=lsp-mode=]] to provide richer, IDE-like capabilities.
+To use this layer with lsp, you must do the following:
+
+1. Add =lsp= to =dotspacemacs-configuration-layers=.
+2. Install =dockerfile-language-server-nodejs= by running
+   =npm i -g dockerfile-language-server=
```

Comment:
I believe you meant
`npm i -g dockerfile-language-server-nodejs`
Because the original didn't work for me.

Reply:
Thanks! I'll fix it.